packages/local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -27,7 +48,7 @@ module.exports = [
   {
     command: '--dev [boolean]',
     description: 'If false, warnings are disabled and the bundle is minified',
-    parse: val => (val === 'false' ? false : true),
+    parse: (val: string) => (val === 'false' ? false : true),
```

Comment:
Maybe instead of `(val: string) => (val === 'false' ? false : true)`, you could do `!!JSON.parse(val)`